README.md

```diff
@@ -32,8 +32,7 @@ fastify.withTypeProvider<TypeBoxTypeProvider>()
 
 ```ts
```

Comment:
Maybe include somewhere in this file that this package now exports all the typebox?

Reply:
Have added a "Overview" section to the readme which mentions that the provider re-exports TypeBox's `Type.*` builder, and includes some extra information about the Fastify additional utility types and TB compiler infrastructure also offered by the Provider.